quarto/src/Controller/GameController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace App\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use App\Api\GameApi;
+use App\Entity\Game;
+use App\Repository\GameRepository;
+class GameController extends Controller {
+  public const GRID_SIZE = 4;
+  private $api;
+  private $twig;
+  public function __construct(\Twig_Environment $twig, GameApi $gameApi, GameRepository $gameRepository) {
```

Comment:
You can create a kind of "GameManager" which is an higher level of abstraction on top of gameApi and gameRepository.

You must have the minimum logic as possible in your controller